hal/src/sercom/v2/spi.rs

```diff
@@ -45,9 +45,9 @@
 //! type Pads = spi::Pads<Sercom0, IoSet1, Miso, NoneT, Sclk>;
 //! ```
 //!
-//! Alternatively, you can use the [`PadsFromIds`] alias to define a set of
-//! `Pads` in terms of [`PinId`]s instead of `Pin`s. This is useful when you
-//! don't have `Pin` aliases pre-defined.
+//! Alternatively, you can use the `PadsFromIds` alias (not available for
```

Comment:
Could you add a `cfg`-based link to `PadsFromIds`? I don't like breaking the link for the narrow SAMD11 use case.